src/dashboard/frontend/app/browse/browse.controller.js

```diff
@@ -78,7 +78,7 @@ class BrowseController {
   // Determines whether a given file can be added to the transfer,
   // depending on the transfer type.
   file_can_be_added(file) {
-    if (this.transfer.type === 'zipped bag' || this.transfer.type === 'zipfile') {
+    if (this.transfer.type === 'zipped bag' || this.transfer.type === 'zipped package') {
```

Comment:
I would have liked to find a way to reference the other constants somehow, or at least reduce both JavaScript, and the HTML constants to one.